mpas_analysis/ocean/nino34_index.py

```diff
@@ -203,18 +247,28 @@ def compute_nino34_spectra(nino34Index):  # {{{
 
     Last Modified
     -------------
-    03/23/2017
+    04/09/2017
     """
 
     # Move nino34Index to numpy to allow functionality with scipy routines
     ninoIndex = nino34Index.values
     window = signal.tukey(len(ninoIndex), alpha=0.1)
     f, Pxx = signal.periodogram(window * ninoIndex,
-                                1.0 / constants.sec_per_month,
-                                scaling='spectrum')
-
-    # computes power spectra, smoothed with 5 point running mean
-    pxxSmooth = _running_mean(pd.Series(Pxx))
+                                1.0 / constants.sec_per_month)
+
+    # computes power spectra, smoothed with a weighted running mean
+    nwts = max(1, int(7*len(ninoIndex) / 1200))
```

Comment:
Can you please provide a reference for this windowing function if it isn't from http://www.cesm.ucar.edu/working_groups/CVC/cvdp/code.html?